config.toml

```diff
@@ -12,6 +12,10 @@
   AuthEnabled = true
   # DisableJoinQuitMessages specifies if join/quit messages should be broadcast when players join the server.
   DisableJoinQuitMessages = false
+  # MuteEmoteChat specifies if the player emote chat should be muted or not.
+  MuteEmoteChat = false
+  # EnableLocatorBar specifies if the locator bar should be enabled or not.
+  EnableLocatorBar = false
```

Comment:
I'd remove this since the actual functionality isn't there. We can add it at a later date